frame/elections-phragmen/src/benchmarking.rs

```diff
@@ -401,15 +401,13 @@ benchmarks! {
 
 		let _ = fill_seats_up_to::<T>(m)?;
 		let removing = as_lookup::<T>(<Elections<T>>::members_ids()[0].clone());
-	}: {
-		assert_eq!(
-			<Elections<T>>::remove_member(RawOrigin::Root.into(), removing, false).unwrap_err().error,
-			Error::<T>::InvalidReplacement.into(),
-		);
-	}
+		let who = T::Lookup::lookup(removing.clone()).expect("member was added above");
+	}: remove_member(RawOrigin::Root, removing, false)
```

Comment:
we lost the check for the returned value of the dispatch.

Also I don't think it works, because it will probably say the benchmark is failing due to an error being returned.

Reply:
Yes I know, I will revert it to manual syntax.